test/files/secure-forward.conf

```diff
@@ -11,7 +11,7 @@
   <buffer>
     @type file
     path '/var/lib/fluentd/secureforwardlegacy'
-    queued_chunks_limit_size "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
+    queue_limit_length "#{ENV['BUFFER_QUEUE_LIMIT'] || '1024' }"
```

Comment:
should this be `32` as well?

Reply:
This is used only for an E2E test, so basically arbitrary data here.

Reply:
To be precise the file is used for an unmanaged fluentd receiver to test log forwarding.

Reply:
ack, thanks